### PR TITLE
Add an option to trust headers set by reverse proxies

### PR DESCRIPTION
**Pull request by carnage**
eg Cloudflare, kubernetes ingresses etc

Going to tidy up and test this tomorrow, putting in a PR as a reminder

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -252,6 +252,10 @@ mail:
     encryption: tls
     auth_mode: ~
 ```
+### [Running behind a trusted proxy](#run-trusted-proxy)
+
+If you are running open cfp behing a proxy server which adds X-Forwarded-For headers (this could be a cloud based load balancer or a service such as cloudflare) you will need to set the environment variable TRUST_PROXIES to true this will ensure that opencfp trusts the headers set by these proxies for the original IP address and ssl mode. Setting this will trust these headers regardless of where the original request originates, so it's advisable to either lock down your instance so that only the trusted proxy can access it, or modifiy the list of trusted proxies in the index.php file to only include the ip addresses of your proxies.
```
Comment (chartjes) [inline README.md]: The project name is OpenCFP

Comment (drbyte) [reply README.md]: Also: `modifiy` should be `modify`

Comment (localheinz) [reply README.md]: Apologies for chiming it, but I think it also should be `Cloudflare`, see https://www.cloudflare.com.

**Comment by localheinz** [comment]
@carnage 

Can you take another look, please?

**Comment by carnage** [comment]
Yep, just haven't had time yet

**Pull request merged**